cli/tsc/dts/lib.deno.unstable.d.ts

```diff
@@ -810,6 +809,30 @@ declare namespace Deno {
    * @category Network
    */
   export interface DatagramConn extends AsyncIterable<[Uint8Array, Addr]> {
+    /** Joins an IPv4 multicast group. */
+    joinMulticastV4(
+      address: string,
```

Comment:
Not sure if it would be useful or just a hindrance, but these types could use string template literal types to define acceptable addresses.

Reply:
Yeah. IPv4 address could be typed (in a rough way) like this:
```ts
type IPv4Address = `${number}.${number}.${number}.${number}`;
```
This doesn't check that the numbers are between 0 and 255 but it's better than nothing.

That being said, this may just be a nuisance in some cases, like f.ex. reading an address from a JSON configuration file, as TS will just type the address as `string`.

Reply:
we should just leave it as string